tests/utils.go

```diff
@@ -76,6 +78,17 @@ func PanicOnError(err error) {
 	}
 }
 
+// Gets an instance of a kubernetes client that includes all the CDI extensions.
+func GetCDIClient() *clientset.Clientset {
+
+	cfg, err := clientcmd.BuildConfigFromFlags(master, kubeconfig)
+	PanicOnError(err)
```

Comment:
panic vs Fail?

Reply:
So the way I read panic vs Fail, is that panic is 100% fatal, while Fail is not good, other tests might still work.

Reply:
Not quite sure what GetCDIclientOrDir() means, the OrDir part is confusing me. Is it a typo and should be OrDie?

Reply:
Sorry. Fat finger on my part. Yes I meant `GetCDIclientOrDie`